From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: John Kleinschmidt <jkleinsc@electronjs.org>
Date: Tue, 11 May 2021 21:35:39 -0400
Subject: chore: allow overriding of enable_pak_file_integrity_checks

enable_pak_file_integrity_checks is not currently overridable
but Electron does not need to run this.

diff --git a/chrome/browser/buildflags.gni b/chrome/browser/buildflags.gni
index 8c19e707371c7ad4fbcd46d58b147de3bfc89790..17c82c078a2beb8146c39290aa37703c7a3b07f2 100644
--- a/chrome/browser/buildflags.gni
+++ b/chrome/browser/buildflags.gni
@@ -21,11 +21,13 @@ declare_args() {
   # app_session_service and some not.
   enable_app_session_service =
       !(is_chromeos_lacros || is_chromecast || is_android)
-}
 
-# If true, the resource .pak files will be hashed and the digest will be
-# embedded in the binary and checked at run-time. This is incompatible with
-# enable_resource_allowlist_generation on Windows because it creates a circular
-# dependency with chrome_dll.
-enable_pak_file_integrity_checks =
-    !(enable_resource_allowlist_generation && is_win) && !is_android
+
+  # If true, the resource .pak files will be hashed and the digest will be
+  # embedded in the binary and checked at run-time. This is incompatible with
+  # enable_resource_allowlist_generation on Windows because it creates a circular
+  # dependency with chrome_dll.
+  enable_pak_file_integrity_checks =
+      !(enable_resource_allowlist_generation && is_win) && !is_android
+
+}
\ No newline at end of file
